test/cmd/deployments.sh

```diff
@@ -30,7 +38,25 @@ os::cmd::expect_success_and_text 'oc env dc/test-deployment-config OTHER=foo -o
 os::cmd::expect_success_and_text 'echo OTHER=foo | oc env dc/test-deployment-config -e - --list' 'OTHER=foo'
 os::cmd::expect_success_and_not_text 'echo #OTHER=foo | oc env dc/test-deployment-config -e - --list' 'OTHER=foo'
 os::cmd::expect_success 'oc env dc/test-deployment-config TEST=bar OTHER=baz BAR-'
-
+os::cmd::expect_success_and_not_text 'oc env -f test/integration/fixtures/test-deployment-config.json TEST=VERSION -o yaml' 'v1beta3'
+os::cmd::expect_success 'oc env dc/test-deployment-config A=a B=b C=c D=d E=e F=f G=g'
+os::cmd::expect_success_and_text 'oc env dc/test-deployment-config --list' 'A=a'
+os::cmd::expect_success_and_text 'oc env dc/test-deployment-config --list' 'B=b'
+os::cmd::expect_success_and_text 'oc env dc/test-deployment-config --list' 'C=c'
+os::cmd::expect_success_and_text 'oc env dc/test-deployment-config --list' 'D=d'
+os::cmd::expect_success_and_text 'oc env dc/test-deployment-config --list' 'E=e'
+os::cmd::expect_success_and_text 'oc env dc/test-deployment-config --list' 'F=f'
+os::cmd::expect_success_and_text 'oc env dc/test-deployment-config --list' 'G=g'
```

Comment:
not sure, would like a way to check that multiple things exist in the output without caring about order (or really, about grep flags)